packages/components/view-switcher/src/view-switcher-button.tsx

```diff
@@ -0,0 +1,59 @@
+import {
```

Comment:
Let me pin it here - I noticed this error in the console, perhaps it's nothing serious, but I guess it's worth checking:
<img width="516" alt="Screenshot 2022-05-20 at 10 43 56" src="https://user-images.githubusercontent.com/49066275/169490515-d95022e9-5aca-4b4f-9f3a-7e9d09639315.png">

Reply:
Thanks for the feedback. I'm still trying to figure out how to solve the problem. 
If you have any clue, let me know.

Reply:
It seems to be caused by the conditional styling:
```js
if (isDisabled) {
    return [
      baseStyles,
      css`
        background-color: ${vars.colorAccent98};
        color: ${vars.colorNeutral60};
      `,
    ];
  }
  if (isActive) {
    return [
      baseStyles,
      css`
        background-color: ${vars.colorNeutral95};
        box-shadow: ${vars.shadow9};
      `,
    ];
  }
```

Reply:
Thanks. It is fixed.